bundles/org.openhab.voice.picotts/src/main/java/org/openhab/voice/picotts/internal/PicoTTSService.java

```diff
@@ -23,17 +23,27 @@
 import org.openhab.core.audio.AudioException;
 import org.openhab.core.audio.AudioFormat;
 import org.openhab.core.audio.AudioStream;
+import org.openhab.core.voice.AbstractCachedTTSService;
+import org.openhab.core.voice.TTSCache;
 import org.openhab.core.voice.TTSException;
 import org.openhab.core.voice.TTSService;
 import org.openhab.core.voice.Voice;
+import org.osgi.service.component.annotations.Activate;
 import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
 
 /**
  * @author Florian Schmidt - Initial Contribution
  */
 @Component
 @NonNullByDefault
-public class PicoTTSService implements TTSService {
+public class PicoTTSService extends AbstractCachedTTSService implements TTSService {
```

Comment:
`AbstractCachedTTSService` already implements `TTSService `.
Another option is to rather use 
```
@Component(service = TTSService.class)
```